examples/time_series/MvGaussianRandomWalk_demo.ipynb

```diff
@@ -2,6 +2,7 @@
  "cells": [
```

Comment:
<p>I'm not sure if the scaler will be compatible with the ppc being stored as an xarray dataset, but when doing that, and using meaningful dimension names, the <code>[:, :, 0]</code>  becomes <code>.sel(dim_name="first_label")</code>  or <code>isel(dim_name=0)</code>.  I don't know which names to use though.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>